src/website/src/styles/home.scss

```diff
@@ -104,7 +104,7 @@
     position: absolute;
     bottom: 0;
     left: 612px;
-    background-image: url('../assets/images/clarity_logo.svg');
+    background-image: url('../assets/images/clarity-logo-spring.svg');
```

Comment:
Can we come up with a means to cycle through the logos based on season, with a fallback?

Reply:
Yeah, I don't think that would be too difficult. It could check the current local date is between specified seasons/dates and toggle the background image based on that.